### PR TITLE
Update react quickstart

### PR DESCRIPTION
**Pull request by jmelberg-okta**
## Description:
- Updates the React quickstart to use the new React SDK

### CODE REVIEW

File: _source/quickstart/react/default-example.md [hunk 1]
```diff
@@ -38,221 +38,150 @@ After you have created the application there are two more values you will need t
...
-  );
-}));
-```
 
-### Create the Callback Handler
-In order to handle the redirect back from Okta, you need to capture the token values from the URL. You'll use `/implicit/callback` as the callback URL, and again you'll use our `auth.js` to delegate the callback details to the [Okta Auth JS](/code/javascript/okta_auth_sdk) library.
+In the relevant location in your application, you will want to provide `Login` and `Logout` buttons for the user. You can show/hide the correct button by using the `auth.isAuthenticated()` method. For example:
 
-Create a new component `Callback.js`, and have it handle the `/implicit/callback` route:
+```javascript
```
Comment (lboyette-okta) [inline _source/quickstart/react/default-example.md]: if we say this `typescript` then we get jsx highlighting

File: _source/quickstart/react/default-example.md [hunk 2]
```diff
@@ -38,221 +38,150 @@ After you have created the application there are two more values you will need t
...
-      if (token.idToken) {
-        this.oktaAuth.tokenManager.add('idToken', token);
-      } else if (token.accessToken) {
-        this.oktaAuth.tokenManager.add('accessToken', token);
-      }
-    }
-  }
+const config = {
+  issuer: 'https://{yourOktaDomain}.com/oauth2/default',
+  redirectUri: window.location.origing + '/implicit/callback',
```
Comment (jmelberg-okta) [inline _source/quickstart/react/default-example.md]: origin

File: _source/quickstart/react/default-example.md [hunk 3]
```diff
@@ -38,221 +38,150 @@ After you have created the application there are two more values you will need t
...
 ```bash
-npm install @okta/okta-auth-js --save
+npm install @okta/okta-react --save
 ```
 
-You will need to create a class that encapsulates the interaction with the [Okta Auth JS](/code/javascript/okta_auth_sdk.html) library. This file will expose a `withAuth` method that makes it easy to create [Higher-Order Components](https://facebook.github.io/react/docs/higher-order-components.html) that include an `auth` property.
+### Configuration
 
-To create this file, you will need the values from the OIDC client that you created in the previous step.  You will also need to know your Okta Org URL, which you can see on the home page of the Okta Developer console.
+You will need the values from the OIDC client that you created in the previous step to instantiate the middleware. You will also need to know your Okta Org URL, which you can see on the home page of the Okta Developer console.
```
Comment (lboyette-okta) [inline _source/quickstart/react/default-example.md]: `You will also need to know your Okta Org URL, which you can see on the home page of the Okta Developer console` - you actually just need to know the issuer

File: _source/quickstart/react/default-example.md [hunk 4]
```diff
@@ -38,221 +38,150 @@ After you have created the application there are two more values you will need t
...
+    return this.state.authenticated ?
+      <button onClick={this.props.auth.logout}>Logout</button> :
+      <button onClick={this.props.auth.login}>Login</button>;
   }
 });
 ```
 
+### Update your `App.js`
+
+Finally, instantiate the Secure Router by passing in your configuration into `Security`, and connect your application's paths:
```
Comment (lboyette-okta) [inline _source/quickstart/react/default-example.md]: hmm, nowhere do we mention a Secure Router, so maybe we should rethink the naming

File: _source/quickstart/react/default-example.md [hunk 5]
```diff
@@ -38,221 +38,150 @@ After you have created the application there are two more values you will need t
...
+
+```javascript
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+import { Security, SecureRoute, ImplicitCallback } from '@okta/okta-react';
+import Home from './Home';
+
+const config = {
+  issuer: 'https://{yourOktaDomain}.com/oauth2/default',
+  redirect_uri: window.location.origing + '/implicit/callback',
```
Comment (lboyette-okta) [inline _source/quickstart/react/default-example.md]: origin

**Pull request merged**